src/conversion.rs

```diff
@@ -1,29 +1,6 @@
 //! A trait used to replace as conversion
 
-use std::any::type_name;
-use std::convert::TryFrom;
-
-/// A type cast trait used to replace as conversion.
-pub trait Cast {
-    /// Performs the conversion.
-    #[inline]
-    fn cast<T>(self) -> T
-    where
-        T: TryFrom<Self>,
-        Self: Sized + std::fmt::Display + Copy,
-    {
-        T::try_from(self).unwrap_or_else(|_| {
-            panic!(
-                "Failed to convert from {}: {} to {}",
-                type_name::<Self>(),
-                self,
-                type_name::<T>(),
-            )
-        })
-    }
-}
-
-impl<U> Cast for U {}
+pub use numeric_cast::NumericCast;
```

Comment:
It limits other types casting.

Reply:
Other types can use `.try_into().expect(...)`, `.try_into().unwrap()`, `.try_into().unwrap_or_else(...)`. 
They don't need the trait `Cast`.

Reply:
The initial purpose of cast is to reduce the verbose code like try_into().unwrap_or_else().

Reply:
Is it possible that for num using numeric_cast::NumericCast and others use cast?

Reply:
99% use cases are number conversions. So I think the replacement is ok.

Reply:
> Is it possible that for num using numeric_cast::NumericCast and others use cast?

It needs a nightly feature: specialization.